src/main/java/mate/academy/webapp/model/User.java

```diff
@@ -43,6 +45,8 @@ public class User implements UserDetails {
 
     private String shippingAddress;
 
+    @ToString.Exclude
+    @EqualsAndHashCode.Exclude
     @ManyToMany(fetch = FetchType.EAGER)
```

Comment:
```suggestion
    @ToString.Exclude
    @EqualsAndHashCode.Exclude
    @ManyToMany(fetch = FetchType.LAZY)
```
also not related to this pr.